# RNA Seq project workflow
## Experiment Overview
The overall goal of this project is to investigate how the presence of abscence of thiamine (Vitamin B) affects gene transcription in *Candida albicans*, an opportunistic fungal pathogen. Three isolates of wild type *C. albicans* (WTA, WTB, and WTC) were grown under two experimental treatments: in the presence of thiamine (thi+) and in the abscence of thiamine (thi-). Next, RNASeq was used to sequence the transcriptome of each of the isolates under each condition. This generated 6 sets of sequencing data (WTA1, WTA2, WTB1, WTB2, WTC1, and WTC2) labeled 1 for the presence of thiamine (1 = thi+) and 2 for the abscence of thiamine (2 = thi-). Each set of sequencing data generated 2 fasta files for the forward and reverse reads (for example: WTC1_1.fq.gz contains the forward reads for isolate C under thi+ conditions and WTC1_2.fq.gz contains the reverse reads).
## Raw data 
I worked on the RNA sequence data (generated by RNASeq) for isolate C in the presence of thiamine (WTC1). The raw data files are WTC1_1.fq.gz (fwr) and WTC1_2.fq.gz (rev).
## Quality analysis and trimming
Fastqc software was used to evaluate the quality of the reads and looked at results (Fastqc/WTC1_1_fastqc.html and Fastqc/WTC1_2_fastqc.html) to determine optimum trimming setting. The number of reads in each fasta file (for forward and reverse reads) was 20752795 before cleaning. 
Then, the reads were trimmed according to settings in vartrimmomatic_script.SBATCH.
After trimming, the fastqc analysis was repeated on the trimmed files to confirm improvements in per-base sequence quality, per-base sequence content, and adapter contamination (Fastqc/WTC1_1.trPE_fastqc.html and Fastqc/WTC1_2.trPE_fastqc.html). Fastqc generated files for paired-end (PE) and single-end (SE) reads, and the paired-end reads were used for further analysis. The number of reads after trimming was 19924384, indicating that 96% of the reads were retained in the cleaning process. See https://docs.google.com/spreadsheets/d/1AOa-XaTzR_PKMIRQDmu8oDTmawXXnkIwEjKOQkNC7Vs/edit?gid=0#gid=0 for details on the quality analysis and trimming of the other files for the other isolates and experimental conditions in this project. 
## Alignment
Trimmed reads were aligned to the reference C. albicans genome (C. albicans genome/ GCF_000182965.3_ASM18296v3_genomic.fna.gz from NCBI) using bowtie2. This generated the output file WTC1.sam. 
The results of the alignment as summarized in the z01 file: 19924384 reads; of these:
  19924384 (100.00%) were paired; of these:
    936104 (4.70%) aligned concordantly 0 times
    17783228 (89.25%) aligned concordantly exactly 1 time
    1205052 (6.05%) aligned concordantly >1 times
    ----
    936104 pairs aligned concordantly 0 times; of these:
      345379 (36.90%) aligned discordantly 1 time
    ----
    590725 pairs aligned 0 times concordantly or discordantly; of these:
      1181450 mates make up the pairs; of these:
        718444 (60.81%) aligned 0 times
        414129 (35.05%) aligned exactly 1 time
        48877 (4.14%) aligned >1 times
98.20% overall alignment rate.
The alignment rates and further details about the alignment results for the other files in this experiment are summarized in https://docs.google.com/spreadsheets/d/1fa-FXVMlCXOZkbHSx_mMg0OXLMy9BeBJg8uWrEMpKGo/edit?usp=sharing. 
## Sorting and Indexing
Samtools was used to convert the mapped reads from the .sam file (sequence aligntment map) to a sorted and indexed .bam file for further analysis. The file was sorted according to the default coordinate (the leftmost coordinate). Then, the file was indexed for fast random access to generate a BAI file (WTC1.srt.bam.bai). 
## Counting Reads
HtSeq-count was used within the HTSeq python package to count how many reads align to each annotated feature in the *C. albicans* genome. This program required loading a conda environment and counts the number of reads from WTC1.srt.bam that are assigned to each gene annotated in the *C. albicans*_GTF annotation file. The reads were counted according to the settings in script/varhtseq.SBATCH. The reads were ordered by position (-r pos), the read were not stranded (-s no). the feature type analyzed was exons (-t exon), and the GTF attribute used was the gene ID (-i gene_id). This generated the output file WTC1_htseqCount. 
## Differential Expression Analysis
Differential expression of *C. albicans* genes between the two experimental conditions (Thi- = NTH, Thi+ = TH) was analyzed using DESeq2. The analysis was performed on the HtSeq output files for each isolate under each condition (NTH_WTA2_htseqCount, NTH_WTB2_htseqCount, NTH_WTC2_htseqCount, TH_WTA1_htseqCount, TH_WTB1_htseqCount, and TH_WTC1_htseqCount). The R script scripts/calb_DESeq_script_FINALR was used in R Studio to run the DESeq2 analysis. This script filtered the HtSeqCount data to include only genes with greater than 10 aligned reads in each sample and set the reference condition as Thi+, meaning that fold change values refer to the change in the expression of the gene when grown in the absence of thiamine. 
### Plots
Quality control was performed and visualized with a principal component analysis (PCA) plot. This stabilizes the variance in genes with low read counts and includes the 500 genes with the most variance. The plot shows clustering by treatment. ![TH-vTH+_pcaplot](https://github.com/user-attachments/assets/7fb89828-be01-481f-b717-f7856e33595c) Next, a DEG analysis was used to identify the genes with the greatest fold change in the thi- condition and the statistical significance of the change in expression. A volcano plot visualizes these results by plotting the Log2 of the fold change against the Log10 of the p value, with cutoffs set at a p value of 0.05 and a log2 Fold Change of 1 (FC = 2). ![image](https://github.com/user-attachments/assets/d1d52b64-1ae6-4af4-98ae-7fbfcd8b2563) 
The volcano plot shows 13 significantly differentally expressed genes (labeled in red) that meet the cutoff critera for both fold change and p value.
### Summary tables
The DEG analysis generated a summary table (DESeq/calb_TH-_v_TH+_allgenes) of the base mean expression level, Log2 Fold Change, p value, and adjusted p value for every gene in the *C. albicans* genome. The summary table DESeq/signif_TH-vTH+ includes this information for the 13 genes identified as significantly differentially expressed. In addition, the GTF annotation file was referenced to include the NCBI Gene ID, gene name, and annotation for each of the 13 genes. This summary table also includes a brief description of the functions of the differentially expressed genes using information from the candida genome database. 
## Gene Ontology Enrichment Test
### Analysis settings
A gene ontology enrichment test was performed on the 13 significantly differentially expressed genes using Pantherdb.com. The analysis type was a PANTHER Overrepresentation Test and the analysis version and release data was GO Ontology database DOI: 10.5281/zenodo.12173881 Released 2024-06-17. The analyzed list was the 13 genes in the signif_TH-vTH+ summary table and the reference list was all Candida albicans genes in database. The annotation data set was GO biological process complete, the test type was Fisher's Exact, and the correction was calculate false discovery rule. 
### Results
The results of the GO enrichment test are summarized in the table GOtest. The significant genes included 4 genes in the thiamine biosyntheic process comapred to an expected 0.01 genes, representing greater than 100 fold enrichment and a p value of 6.47 E-11. This includes significant enrichment in processes such as primary alcohol biosynthesis, thiamine metabolism, and thiamine-containing compound biosynthesis. The results also showed significant enrichment for pyridoxine and pyroxidal phosphate biosynthesis. Pyoxidine (Vitamin B6) is another type of Vitamin B. In addition, the biological process of ergosterol biosynthesis was enriched in the set of genes. Ergosterol is a sterol that is an important fungal metabolite.
## Overall Interpretation
